src/figuresClasses.ts

```diff
@@ -1,19 +1,95 @@
-export interface Figure {
-
+enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle',
 }
```

Comment:
If you use enum, please, look at checklist (Enum constant names usually start with capital letter. Values can have any case that's necessary.)

Reply:
Ohh... I knew it and for some reason didn't use it.